.travis.yml

```diff
@@ -1,235 +1,194 @@
+  install:
+    - sudo sysctl -w vm.max_map_count=262144
+  script:
+    - docker-compose -f .ci/$DOCKER_COMPOSE_TEST.yml run kuzzle
+
+# Documentation deployment
+_doc-deploy: &doc-deploy
+  stage: Deployments
+  language: node_js
+  node_js: 10
+  env: &doc-deploy-env
+    - NODE_ENV=production
+    - AWS_DEFAULT_REGION=us-west-2
     - AWS_ACCESS_KEY_ID=AKIAIYAXFUAHXOWP2MJA
     # AWS_SECRET_ACCESS_KEY
-    - secure: "Hg5X8SsJTG0r/ge4IM7eDG6Q6t1S/aG8aO0mOwEabdBRNgbJJgT9mSv2Ukdxuse0kUl/OyA7/51pOf4tYHgRlupGzaJU9SzDTMjapYLvg1a8EhE0Fs68zOs1EGPRktnLHHKot5hnwIB1i4PKKYlooF3n4+kTX9Rj1F4KAjoroRB5kx0HOqGDJbUkifyqjrI8e0j8sNeHtrcf2PaYgylTgx8iQEsaAr5v/ba/k9Z8E5rikDqXtroHvPNCSqblJvzDLOuX4NyHqfT38sc/mBNSREImffjN8nPkMYXo0nYrKXEiXbeCeGCqFqEM+Psgjk1BcUREpfAOKBxQYTrnpz6Ga+/D+KVzGo/FPF/QyapgAokGdj9rUAuytsoPabz7tqBpNf/4ZS0SKlh9PZjgQu5CvTScDeQnavS48ChdpWnrkrmFx9DpaYp1lBFxiycUz1Z3bpLaA8odey7lBXwj2L5gpht3BpuxH2+kxb699TMiXSdYb/XOjvIWd9Ei6MkRHAmphlvEJE/sThsA9k1DdYQj05rbCQYfqEv1WutdyHc8FtQVlTivZ4dWJ/O+xuL4aOB7Lryfm2HdK9631pQVNtBbTAD3I1xwc/GzdVBLlZu8AK5vVVXie2m0bAcPuLLRarqjAaSFr3dY228OJWWSjdBgeVOQdWQ/SavPbRHU94KrdmA="
-
-
+    - secure: h8n58Siclba6vRZEUcVzH2UTCBaA2src/mJvPaiXln1dPP4+/CICkfoljggA/qZDv9pv1ck07u7fHf6EJa7ONO2aNKD/r9CyyKHbZsgT5Wz6zLaOJfu/DmLZq9yclG/e9XmlK7qpjW+H7vCPn98GNVaU2h8cjqNVZ3KpkLwQ0j2RBoM6x4jpOFUsbiwnKWaHxS2u3JHpZ/L8PM9uy2JDoRm52HW5k+fmvyxwNis+R5ezteJJYcqhYhun9CRDVjGg47t8YFkO+fXjtgl5EAhNlOtE1UixV7h8+MVbmS0XJQm4Xen76J0a2O3xEGOggsdMJebTuDlHk97jhR17pMFu0lfM4V2YmiC8BclY/fdnxX7aoc9hPi9Zf6ReHOqVo8jWP4ArS7awHdKW8c5zkH2itf7M6tw89hnDmoXVk/48joOyj+vMwhvI2wJ3oUjhsjpZWG7Mks4hY2CYmkA5n56K/6+qgPq19qZr4MKqW1zr8066TNzqa2ZtdbtSzIyP0jcXLyNOxhwXejyBnHTyJ7ZVbiNxyGTvLQcOP1UYNn1LxmDTJWvDCguXbwLQ2oHzD032xVuNCS2XjZ1dDFtEtEfDniVnOGosEzJWK2kkRCaOzW5GT+VunfZnN9j4MNZVfI1iCW3EQ4CI7n/Ct89y4YrezwRoFjvMQvhb64pVLwMeOp4=
+  cache:
+    npm: true
+    directories:
+      - $HOME/.cache/pip
+      - doc/framework/node_modules
+  addons:
+    apt:
+      packages:
+        - python
+        - python-pip
+  install:
+    - pip install awscli --upgrade --user
+  script:
+    - npm run doc-prepare
+    - npm run doc-build
+  deploy:
+    provider: script
+    script:
+      - npm run doc-upload
+    skip_cleanup: true
+  after_deploy:
+    - npm run doc-cloudfront
+
+# ------------------------
+# Jobs configuration
+# ------------------------
 jobs:
   include:
-
-# # ---------------------------------------
-# # Unit Tests & Linters
-# # ---------------------------------------
-
+    # ---------------------------------------
+    # Unit Tests & Linters
+    # ---------------------------------------
     - stage: Unit Tests & Linters
-      if: type = pull_request OR (type = push AND branch =~ ^master|.*-dev|.*-stable$) OR type = cron
       name: Unit Tests & ESLint
       os: linux
       sudo: required
       language: node_js
-
-      cache: npm
-
-      node_js:
-        - "8"
-
+      node_js: 10
+      cache:
+        npm: true
+        directories:
+          - node_modules
```

Comment:
why? `$HOME/.npm` is already cached and should be good enough, no?

Reply:
https://github.com/kuzzleio/kuzzle/pull/1394#discussion_r315646799